siteorigin-panels.php

```diff
@@ -925,7 +928,8 @@ function siteorigin_panels_render( $post_id = false, $enqueue_css = true, $panel
 		}
 	}
 
-	if( is_rtl() ) $panels_data = siteorigin_panels_make_rtl( $panels_data );
+	// A filter to change the panels_data right before the content is rendered.
+	$panels_data = apply_filters( 'siteorigin_panels_data_before_render', $panels_data, $post_id );
```

Comment:
Shouldn't this filter now be added somewhere to do whatever `siteorigin_panels_make_rtl` was doing?